### ISSUE TITLE
Logseq：Format of date variables and pages name according to that

### ISSUE THREAD

**Issue by yoghurtdewdew** [omnivore-app/logseq-omnivore#127]
Does the date variables support custom formatting? 

I would like pages like “Omnivore-YYYY-MM” or “Omnivore-YYYY-W33”, rather than a single page or multiple pages.

**Comment by sywhb** [comment]
Hey @yoghurtdewdew , sorry for the late reply.

So far we only support custom date format in the page name by changing the "Preferred date format" in the Editor Settings but this is good to add and I can try to add that.

<img width="431" alt="Screenshot 2023-12-08 at 1 34 05 PM" src="https://github.com/omnivore-app/logseq-omnivore/assets/12963965/2c3d878a-57f6-4380-b8b2-841b87e785f0">

**Comment by yoghurtdewdew** [comment]
<img width="453" alt="image" src="https://github.com/omnivore-app/logseq-omnivore/assets/63794438/553c9dce-1024-4fc8-ba81-0e1de70d3477">
<img width="468" alt="image" src="https://github.com/omnivore-app/logseq-omnivore/assets/63794438/408336b8-4a84-476f-907e-126e6673f070">

To sync use a single page or multiple pages maybe not easy to see them, so multiple pages use  “Omnivore-YYYY-MM” or “Omnivore-YYYY-W33” will be great.

**Comment by sywhb** [comment]
Yeah, it makes sense. Probably we can automatically sync into multiple pages if there is a dynamic page name in the settings.

Before that, you can also just put the date variable `{{{ date }}}` in the "pageName" settings plus the "preferred date format" to achieve what you want.

<img width="413" alt="Screenshot 2023-12-12 at 10 52 09 AM" src="https://github.com/omnivore-app/logseq-omnivore/assets/12963965/ae22e519-08d8-4b63-8b5f-a9f47f9b4ccc">

**Comment by yoghurtdewdew** [comment]
That didn't solve my problem. Prefered date format surely will be one day & time to know the date & time for each webpage. But pageName I would prefer monthly & weekly pages.

It means, if I set Prefered date format to be YYYY-MM-DD, I get too much pages. If I set Prefered date format to be YYYY-MM, I could not get the day & time of each webpage.
